### DISCUSSION TITLE
Theme Installation Failed - Any Suggestions?

### DISCUSSION THREAD

**Discussion by kodekatz** [nusserstudios/tailbliss#142] [Q&A]
Hi, I am interested to learn Hugo with your theme, it surely looks great. But when I am trying to run this it showed the following:

[hugo] ERROR Failed to read Git log: fatal: not a git repository (or any of the parent directories): .git
[tw] 
[tw] Rebuilding...
[hugo] Built in 297 ms
[hugo] Error: error building site: POSTCSS: failed to transform "css/style.css" (text/css): 'C:\Program' is not recognized as an internal or external command,
[hugo] operable program or batch file.
[hugo] npm run watch:hugo exited with code 1

I am new to Hugo, which one is wrong? Can you show me suggestions? Thanks

**Comment by nusserstudios** [comment]
1. What OS are you using right now?
2. What version of Node.js and NPM are you using?

Please provide as much info as possible, if you can.
